en/advanced_config/esc_calibration.md

```diff
@@ -1,32 +1,30 @@
-# ESC calibration
+# ESC Calibration

```

Comment:
@bkueng OK, I redid this page to make it more clear. 

My understanding is that all calibration does is determine and the correct PWM MIN and MAX for the ESCs. For high quality ESCs you can trust the info on their specs for these values, but for low quality ESC you need to calibrate to find these values. 
1. Is that correct?
2. Is it correct that there is only one stored ESC PWM min/max value? The implication being that you must the same model of ESC for all motors and all will have same max/min values?
3. How does the calibration work? Ie I assume that the calibration ramps up PWM until motor movement starts to get PWM_MIN and then continues until increasing PWM does not result in greater speed to get PWM_MAX ... but what is it actually measuring to get that speed range?

Further more, we don't say anything much about ESC selection anywhere else. We should probably have a section on that in our assembly guide. This PR might not be the right place but:
1. Do we support all types of ESCs? 
   * Are there types we do/don't support
   * Are there types we do/don't recommend
1.  I understand that some ESCs use different firmware. What types of firmware do we support recommend?
1. Are all ESCs controlled as PWM ie via MAIN or are some off other bus?
1. Do all ESC require calibration - ie if off another bus?